### DISCUSSION TITLE
What file is the "dataset" referred to in the README.md?

### DISCUSSION THREAD

**Discussion by Alple228** [egorsmkv/speech-recognition-uk#35] [Q&A]
My main question is in the title. I copied these steps below so there's a little more context:
"
1. Download Ukrainian dataset from [https://github.com/egorsmkv/speech-recognition-uk](https://github.com/egorsmkv/speech-recognition-uk).
2. Delete Common Voice folder in dataset
"
I am not sure which file is the dataset. I downloaded "speech-recognition-uk-master.zip" as that was the only download I could find and I couldn't find anything that was obviously a dataset. Also, I cited step 2 above because there was no "Common Voice folder" in the folder I downloaded. Could somebody point in the right direction. I am trying to transcribe recordings of my constructed language for a speech processing class project. (My conlang happens to share a lot of sounds with Ukrainian). Any help would be appreciated!

**Comment by egorsmkv** [comment]
Here it is - https://github.com/egorsmkv/speech-recognition-uk?tab=readme-ov-file#-datasets
